pkg/reconciler/pingsource/jobrunner/pingsource_test.go

```diff
@@ -184,8 +174,8 @@ func TestAllCases(t *testing.T) {
 	}
 
 	logger := logtesting.TestLogger(t)
-	reporter := &mockReporter{}
-	ce := kncetesting.NewTestClient()
+	reporter := &MockStatsReporter{}
+	ce := adaptertesting.NewTestClient(reporter)
```

Comment:
It is now the job of the adapter to test that metrics are collected or not, not the job runner.